scripts/shared/nodeinfo/nodeinfo.json

```diff
@@ -0,0 +1,11 @@
+{
+    "location": {
+        "continent": "NA",
+        "country": "Canada",
+        "region": "Ontario",
+        "municipality": "Toronto",
+        "uri": "https://tomesh.net"
+    },
+    "software": {
+        "repo": "prototype-cjdns-pi",
+        "branch": "develop",
```

Comment:
This will also need to be changed when develop gets merged into master.

Reply:
Changing this when being merged will be hard. Instead consider:

repo ` git remote get-url origin | awk  -F / '{print $5}'|  cut -d '.' -f1`
branch: `git branch | grep \* | cut -d ' ' -f2` or  `git rev-parse --abbrev-ref HEAD`
maybe also add current commit `git rev-parse HEAD`

Reply:
Will do.